src/relax/transform/static_plan_block_memory.cc

```diff
@@ -107,7 +107,8 @@ class StorageToken : public ObjectRef {
     }
 
     ObjectPtr<StorageTokenNode> n = make_object<StorageTokenNode>();
-    n->bytes = (size * dtype.bits() * dtype.lanes() + 7) / 8;
+    int bytes_per_element = (dtype.bits() * dtype.lanes() + 7) / 8;
+    n->bytes = size * bytes_per_element;
```

Comment:
Thanks for fixing! Credit to @Ubospica, we found that we can directly use `size * dtype.bytes() * dtype.lanes()`, which is equivalent with your fix.